ui/app/pages/add-token/util.js

```diff
@@ -9,5 +7,5 @@ export function checkExistingAddresses (address, tokenList = []) {
     return existingToken.address.toLowerCase() === address.toLowerCase()
   }
 
-  return R.any(matchesAddress)(tokenList)
+  return tokenList.some(matchesAddress)
```

Comment:
This was migrated to another location in #8868 - and also converted to not use ramda.